runtime/autoload/provider/python3.vim

```diff
@@ -1,45 +1,16 @@
-" The Python3 provider uses a Python3 host to emulate an environment for running
-" python3 plugins. :help provider
-"
-" Associating the plugin with the Python3 host is the first step because
-" plugins will be passed as command-line arguments
-
 if exists('g:loaded_python3_provider')
   finish
 endif
-let [s:prog, s:err] = provider#pythonx#Detect(3)
-let g:loaded_python3_provider = empty(s:prog) ? 1 : 2
+let g:loaded_python3_provider = 1
 
-function! provider#python3#Prog() abort
```

Comment:
Can we deprecate this function first and then remove? A skeleton like vimscript that only contains the (public) autoload functions, e.g. `provider#python3#Prog()` `provider#python3#Error()` etc. are left but simply calls the lua counterpart.

Reply:
These functions are not documented and IIUC were meant to be internal implementation. The technology for underscore prefix for private functions didn't exist back then.